packages/mui-material/src/SvgIcon/SvgIcon.js

```diff
@@ -211,6 +240,8 @@ SvgIcon.propTypes /* remove-proptypes */ = {
   viewBox: PropTypes.string,
 };
 
-SvgIcon.muiName = 'SvgIcon';
+if (SvgIcon) {
```

Comment:
I am adding a fix for this separately. But for your changes, the quickest fix would be to use inline css instead of `sx` prop since it's just one property.

[a.patch](https://github.com/mui/material-ui/files/14906382/a.patch)

Reply:
Thanks @brijeshb42! Just applied your suggested changes.